stratum/hal/lib/tdi/dpdk/dpdk_chassis_manager.cc

```diff
@@ -143,7 +143,13 @@ ::util::Status DpdkChassisManager::SetPortParam(
     SetRequest::Request::Port::ValueCase value_case) {
 
   auto& config = node_id_to_port_id_to_port_config_[node_id][port_id];
-  config.SetParam(value_case, singleton_port);
+  ::util::Status status = config.SetParam(value_case, singleton_port);
+  if (!status.ok()) {
+      return MAKE_ERROR(ERR_INVALID_PARAM)
```

Comment:
Why not forward the status returned by SetParam?

Reply:
You are right, that's even better. Changed as per your suggestion